### ISSUE TITLE
Issues with generating polls

### ISSUE THREAD

**Issue by yh570** [trailofbits/cb-multios#91]
Hi, I met some problems with generating polls. I successfully built the challenges with `./build.sh`, and then I tried to generate the poll with `./genpolls.sh`, however, I got many error messages in this process, and also when I ran `./tester.py -a -o out.xlsx` it stuck(overnight) with following challenges:
```
Testing A_Game_of_Chance...
POV:
        Running 2 test(s) => Passed 2/2
POLL:
        for-release:    Running 200 test(s)
```
Attached is the log file with the command `./genpolls.sh &> genpolls.log`. It looks like I didn't generate the polls correctly, but I have no idea how to fix this problem. Please give me some comments about this issue, thank you.

[genpolls.log](https://github.com/trailofbits/cb-multios/files/7476991/genpolls.log)

**Comment by liushengahn** [comment]
Me, too. Have you solved it now?

**Comment by woodruffw** [comment]
I don't believe anyone is currently maintaining the poller scripts, sorry. We don't currently use them internally. If you make any progress, please feel free to submit a PR with fixes or additional documentation for others.
